cypress/integration/loginFlows/userLogin.spec.js

```diff
@@ -32,5 +32,8 @@ describe('User Login', () => {
     // User should be redirected to onboarding
     const { baseUrl } = Cypress.config();
     cy.url().should('equal', `${baseUrl}?signin=true`);
+
+    // User data should exist on the document
+    cy.document().should('have.nested.property', 'body.dataset.user');
```

Comment:
Refactor (blocking): The check for the `data-user` attribute is unnecessary as the application will not work, i.e. e2e tests fail in other areas and this is an implementation detail a user would be unaware of.

The check for the non-existence of the `data-attributer` in the logout test makes sense though even though it's an implementation detail as this is what the bug was.